tests/Unit/Core/Data/AbstractTypedCollectionTest.php

```diff
@@ -48,7 +48,7 @@ public function testConstructor()
 
     public function testInvalidConstructor()
     {
-        $this->expectException(TypeException::class, 'Invalid element type Tests\Unit\Core\Data\InvalidCollectionTestElement, expected Tests\Unit\Core\Data\CollectionTestElement');
+        $this->expectException(TypeException::class);
```

Comment:
Same here, I think a call `expectExceptionMessage` should make the assertion verified